towhee/hub/builtin/operators/milvus_search.py

```diff
@@ -55,13 +56,12 @@ class milvus_search:
     [<Entity dict_keys(['path', 'img', 'vec', 'results'])>,
      <Entity dict_keys(['path', 'img', 'vec', 'results'])>]
     """
-    def __init__(self, collection, **kwargs):
-        from towhee.utils.milvus_utils import Collection
 
-        if isinstance(collection, str):
-            self.collection = Collection(collection)
-        elif isinstance(collection, Collection):
-            self.collection = collection
+    def __init__(self, collection_name, **kwargs):
```

Comment:
We should support both pymilvus collections and collection names here, according to the feedback when we review the code examples with the Milvus team.